bench/trust_token.cc

```
@@ -0,0 +1,347 @@
+  bool BeginIssue(benchmark::State &state) {
+    uint8_t *issue_msg = NULL;
+    size_t msg_len = 0;
+    if (!TRUST_TOKEN_CLIENT_begin_issuance(client_.get(), &issue_msg, &msg_len,
+                                           batchsize_)) {
+      state.SkipWithError("TRUST_TOKEN_CLIENT_begin_issuance failed.");
+      return false;
+    }
+    free_issue_msg_.Reset(issue_msg, msg_len);
+    return true;
+  }
+  void SpeedIssue(benchmark::State &state) {
```

Clang (14.0.6)

[warning] method 'SpeedIssue' can be made static

(readability-convert-member-functions-to-static)